src/examples/raw_events.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+from aw_client import ActivityWatchClient
+from aw_core.models import Event
+
+client = ActivityWatchClient("test-client", testing=True)
+
+# get the buckets
+buckets = client.get_buckets()
+
+bucket_id = "{}_{}".format("aw-watcher-window", client.client_hostname)
+
+# fetches 1000 events from the bucket
+events = client.get_events(bucket_id, limit=1000)
+
+# sum the time spent on each app
+time_spent = dict()
+for event in events:
+    app = event.data["app"]
+    if app in time_spent:
+        time_spent[app] += event.duration
+    else:
+        time_spent[app] = event.duration
+
+print(time_spent)
+
+# Rename app "Code" to "VS Code" in the events 
```

Comment:
I don't think this section belongs in this example. 

Replacing events is something you'd rarely want to do, and we could just refer to the relevant aw-client example for redacting events.

Reply:
Cool we redo this bit.